package.json

```diff
@@ -1,37 +1,50 @@
 {
   "name": "@open-formulieren/formio-renderer",
-  "version": "1.0.0",
+  "version": "0.1.0",
   "description": "Alternative Form.io renderer made for Open Forms.",
-  "author": "",
-  "license": "EUPL-1.2",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/open-formulieren/formio-renderer.git"
-  },
-  "main": "dist/index.js",
-  "module": "dist/index.modern.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/esm/index.d.ts",
   "source": "src/index.tsx",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
```

Comment:
Not opposed to this but I've had some issues in the past with packages defining esm modules for the import format (basically all modern es/js) which worries me a bit. Especially since node_modules libraries aren't always processed by loaders in webpack etc.

I'm also a bit worried if this is going to work at BD because of certain old systems (but I guess they just hit a build artifact).

Reply:
I made the setup the same like design-token-editor which is used with webpack without issues so far. The config should also be that doing `import {Foo} from '@open-formulieren/formio-renderer'` would default to CJS. The JS itself is transformed into es6.

Let me know if we run into any issues, for now both are supplied with minimal effort and I expect no problems.